engines/wintermute/base/base_file_manager.cpp

```diff
@@ -318,7 +318,7 @@ Common::SeekableReadStream *BaseFileManager::openPkgFile(const Common::String &f
 
 bool BaseFileManager::hasFile(const Common::String &filename) {
 	if (scumm_strnicmp(filename.c_str(), "savegame:", 9) == 0) {
-		BasePersistenceManager pm(BaseEngine::instance().getGameTargetName());
+		BasePersistenceManager pm(BaseEngine::instance().getGameTargetName().c_str());
```

Comment:
Might as well update the BasePersistenceManager-constructor to take Common::String& then.